web/src/redcross/blogModule/components/Blog.ng.html

```diff
@@ -1,7 +1,8 @@
 <div class="blog-outter">
-    blog
     <blogPost *ngFor="let p of posts" [post]="p">
     </blogPost>
-    <button class="add-post-button" md-fab (click)="addPost()"><md-icon>add</md-icon></button>
+    <button md-fab class="add-post-button" (click)="addPost()">
```

Comment:
You can add button by using the mat-icon-button with mat-icon nested inside. Something like:
`<button mat-icon-button>
<mat-icon>add</mat-icon>
</button>`
Using mat-icon, Angular Material should automatically load

Reply:
Or, if you really can't figure it out, checkout  Justin's code. He basically put a '+' sign there. 
This is not the best approach, but we can change that later.
His code is already merged to master.